more_services/index.html

```diff
@@ -0,0 +1,619 @@
+      <hr>
+      <div class="grid grid-cols-3">
+      <div class="sidebar mt-16 w-56 ml-52 col-span-1 font-mark font-normal text-base flex-col">
+          <ul class="">
+              <li class="side-nav-item cursor-pointer mb-2">
+                <a href="/home-improvement">
+                  <svg-icon class="icon-container">
```

Comment:
Because of the ml-52 and the fixed width of w-56, the sidbar is overflowing when the viewport with is shrinking. Rather we could just place the sidebar and the right div in the center of the screen by placing the entire grid in the center.